### PR TITLE
Feature/multi arch workflow

### PR DESCRIPTION
**Pull request by Maleware**
This creates a Workflow with buildjet and builds multi-arch product images. Those get pushed in stackable-experimental.

docker buildx build --platform linux/amd64,linux/arm64 is actually not available. However, this command wouldn't make use of the possibility that two different native runners are available. 

This workflow is supposed to leverage the ability to build nativ on amd64 & arm64 and in the end build those two together into a manifest list and push it without the platform specific endings.

### CODE REVIEW

File: .github/workflows/dev_arm.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+name: Publish 0.0.0-dev images
```
Comment (razvan) [inline .github/workflows/dev_arm.yaml:2]:
```suggestion
name: Publish ARM 0.0.0-dev images
```

File: image_tools/bake.py [hunk 2]
```diff
@@ -114,6 +115,11 @@ def bakefile_product_version_targets(
     tags = build_image_tags(image_name, args.image_version, versions["product"])
     build_args = build_image_args(versions, args.image_version)
 
+    if args.multiarch:
+        if args.architecture[0] == "linux/amd64":
+            tags[0] = tags[0] + "-amd64"
+        else:
+            tags[0] = tags[0] + "-arm64"
```
Comment (razvan) [inline image_tools/bake.py:122]: I would move this to the `build_image_tags` function for cleanness.

**Review by razvan — COMMENTED** [review]
Have you seen the actionlint errors that say that `buildjet-4vcpu-ubuntu-2204-arm` is not a valid label?

**Comment by Maleware** [comment]
Outdated, closing this

**Pull request closed without merging**